### ISSUE TITLE
HOWTO is missing?

### ISSUE THREAD

**Issue by miketotman-sd** [rsnapshot/rsnapshot#93]
The HOWTO document that the readme refers to is not present on the main web site. I can see its source in the documentation directory though, but it's not readable easily.

**Comment by bebehei** [comment]
Yes, you're right. The so-called HOWTO not easily readable. It also has been written for a 1.2.x-version. Some things may not apply anymore to recent versions.

I haven't touched the HOWTO yet, but my plan is to convert it to Markdown. To make it more readable in its source and be able to convert it into every other format.

Additionally this has to be enhanced.

When the HOWTO ok, I'll put it onto the Website, too.

**Comment by azrdev** [comment]
then maybe put a short notice "down for maintenance" there instead of the 404, which just animates people like me to look where the howto might have been moved without adjusting all references to it

**Comment by JF002** [comment]
I was searching for the HOWTO too. I hope you'll be able to put it back on the website soon :)

**Comment by bobbyd3** [comment]
I'd like to see the HOWTO back online in a published format documenting the current version of rsnapshot as well. Thanks!

**Comment by mtowara** [comment]
I second to that.
The manpage + standard conf would probably be enough as a quick remedy.

Or just remove the references

**Comment by maticmeznar** [comment]
@bebehei When will the HOWTO be published and linked to?

**Comment by KonstantinSchubert** [comment]
+1

**Comment by crisp-github** [comment]
Howto would be a big help, thanks.

**Comment by matkoniecz** [comment]
> Yes, you're right. The so-called HOWTO not easily readable. It also has been written for a 1.2.x-version. Some things may not apply anymore to recent versions.

And this outdated version is currently located at https://github.com/rsnapshot/rsnapshot/blob/master/docs/docbook/rest.txt

**Comment by olberger** [comment]
Seems archive.org's latest version was : http://web.archive.org/web/20150603083713/http://rsnapshot.org/howto/1.2/rsnapshot-HOWTO.en.html

**Comment by jdpipe** [comment]
I suggest what whatever state the HOWTO is in, it would be better than a 404 error. Perhaps it might be easy to quickly convert what you have already using Pandoc, https://pandoc.org/try/, and then to work on improving it over time, after that.

**Comment by robshep** [comment]
http://www.rsnapshot.org/ mentions:

> "the rsnapshot HOWTO at http://www.rsnapshot.org/"
